lib/belvo.rb

```diff
@@ -110,6 +110,12 @@ def tax_status
       @tax_status = TaxStatus.new @session
     end
 
+    # Provides access to TaxRetentions resource
+    # @return [TaxRetentions]
+    def tax_retentions
+      @tax_retentions = TaxRetentions.new @session
+    end
+
     # Provides access to Instituions resource
```

Comment:
```suggestion
    # Provides access to the Institutions resource
```

I have a to do to review all the code comments in our ruby SDK. Just taking advantage now to get some out of the way ;)